### ISSUE TITLE
Concatenate conditions in PATH operations

### ISSUE THREAD

**Issue by granhal** [simpleidserver/SimpleIdServer#17]
Hello guys,

When trying concatenate conditions in PATH operations, return a error.
Request example:
 ```
{
     "schemas":
      ["urn:ietf:params:scim:api:messages:2.0:PatchOp"],
     "Operations": 
     [
     	{
     		"op":"remove",
     		"path":"members[display eq \"es2112312313201p\" and value co \"es\"]"
     	},
	 	{
         	"op":"add",
         	"path":"members",
         	"value": 
         		[
           		{
             		"display": "James Smith",
             		"refer": "473d93df9210",
             		"value": "473d93df9210"
           		}
         		]
      	}
     ]
}
```
This error is in:
**"path":"members[display eq \"es2112312313201p\" _and value co \"es\"_]"** (note: automatic remove slash... but they are neccesaries)

Error: Internal Server Error
_SimpleIdServer.Scim.Exceptions.SCIMAttributeException: PATCH can be applied only on existing attributes_

Regards.

**Comment by simpleidserver** [comment]
Hello,

There is an issue in the "SCIMRepresentationExtensions" class, in fact the method "GetAttributes" returns an empty list of attributes. We are going to fix this issue and include the fix in the next release.

Regards,

**Comment by simpleidserver** [comment]
The bug is fixed in the release 1.0.6, can-you please try again and replace the parameter "refer" by "$ref"  ? In fact the parameter "refer" is not defined in the "Group" schema.

**Comment by granhal** [comment]
> The bug is fixed in the release 1.0.6, can-you please try again and replace the parameter "refer" by "$ref" ? In fact the parameter "refer" is not defined in the "Group" schema.

This is a problem, in C# it is not possible to create vars with $ or ref, $ is reserved and ref is reserved too ... it is a problem if it is necessary to force the modification of the scheme. That's why I changed the name of the variable.

Thanks for all!

Regards.
